tests/tests_auth.py

```diff
@@ -89,7 +96,47 @@ async def endpoint(request: Request) -> Response:
         self.assertEqual('user', user.username, "Wrong username for user")
 
         self.assertEqual(
-            ['authenticated', 'blueshirt'],
+            ['authenticated', BLUESHIRT_SCOPE],
+            scopes,
+            "Wrong scopes for user",
+        )
+
+
+class FileAuthTests(test_utils.AsyncTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.auth_fixture = Path(__file__).parent / "fixtures/auth-file.yml"
+
+        self.backend = FileBackend(path=str(self.auth_fixture))
```

Comment:
I think it's legit to pass a `Path` to `open`, so we could probably let `FileBackend` accept a union of those if you like?